docs/about/index.rst

```diff
@@ -17,13 +40,156 @@ Code of conduct
+   outlines the planned efforts, some of which have been completed. There are
+   still massive gaps, particularly in the user guides.
+
+   **How you can help:** critique the current content, improve writing and
+   examples, port the current documentation to the numpydoc format.
+
+#. *Improve example notebooks and add new ones*
+   The repository started out with all notebooks in the `notebooks` directory
+   as `.ipynb` files. These notebooks are now being
+   converted into `.py` files that
+   `render properly on the website <https://fairlearn.github.io/auto_examples/notebooks/index.html>`_,
+   while still being downloadable as `.py` or `.ipynb` files.
+   
+   - Some of the notebooks may be better off as parts of the user guide.
+   - As pointed out in various issues the existing notebooks have plenty of
+     potential for improvement to better reflect the sociotechnical focus of Fairlearn.
+     To accelerate this process, we experiment with weekly sessions
+     where people can discuss individual example notebooks in detail.
+   - There are only very few existing example notebooks. To cover a
+     wider range of applications we need to work with stakeholders to develop
+     more guides.
+
+   **How you can help:** critique and improve existing notebooks, join the
+   discussion sessions, create new notebooks, move the notebooks to the user
+   guide.
+
+#. *Nurture a diverse community of contributors*
+   We would like to ensure that all feel welcome and can contribute in a way
+   of their choosing. So far we have monthly developer calls.  We are
+   experimenting with adding deep dive sessions around sociotechnical aspects
+   of notebooks.  We also work with universities on engaging with student
+   contributors through course projects.  Finally, we are working to improve
+   documentation.
+   
+   **How you can help:** reach out with feedback on what is working and what
+   is not working; suggest how to improve things; point out where
+   documentation, our processes or any other aspect of the projects create
+   barriers of entry.
+
+#. *Move towards compatibility with scikit-learn*
+   
+   **How can you help:** help identify non-compatible aspects, improve code
+   towards compatibility.
+
+#. *Add regression to the exponentiated gradient algorithm*
+   The exponentiated gradient algorithm originally worked only for binary
+   classification.  There is a pull request that implements the extension to
+   regression (and the bounded group loss fairness criterion).
+   We intend to add the corresponding documentation before completing it.
+
+   **How you can help:** review and improve the documentation once it's added
+   to the repository
+
+#. *Document and modularize the Fairlearn dashboard*
+   The Fairlearn dashboard is currently not ready for contributions by
+   the community.  Over the next few months, the Microsoft Research and Azure
+   ML teams are working to properly open it for contributions similarly to all
```

Comment:
"will be working"

Reply:
I guess that depends on whether you consider discussions as "work" :-) I do. Also, this version of writing it doesn't need an update shortly.

Reply:
I would say it's more a question of tenses - "next few months" is future, but "are" is present tense.